src/bridge_stream.rs

```diff
@@ -6,17 +6,20 @@ use crate::pcap_util;
 
 use crate::stream::StreamItem;
 use failure::Fail;
+use failure::_core::iter::Peekable;
```

Comment:
Stupid failure exports. This should be `std::iter::Peekable`.

Reply:
derp, thx man